pipeline_steps/pubcloud.groovy

```diff
@@ -237,8 +237,11 @@ def runonpubcloud(Map args=[:], Closure body){
 }
 
 def uploadToSwift(Map args){
-  if(fileExists("${WORKSPACE}/artifacts")){
-    print("WORKSPACE/artifacts directory found, Preparing to upload artifacts.")
+  if (env.RE_HOOK_ARTIFACT_DIR == null) {
```

Comment:
How about `if (env.RE_HOOK_ARTIFACT_DIR) {` to cater for situations where `env.RE_HOOK_ARTIFACT_DIR` is an empty string?